.gitignore

```diff
@@ -11,3 +11,7 @@ vendor/.git
 node_modules
 vendor
 diff.txt
+.phpcs-cache
```

Comment:
This should not be necessary, see:

https://github.com/joindin/joindin-api/blob/f7dede99cbe0c325565d64626999c4797ef4d68f/composer.json#L67